orc8r/gateway/c/common/ebpf/BUILD.bazel

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "ebpf",
+    hdrs = [
+        "EbpfMap.h",
+        "EbpfMapUtils.h",
+    ],
+    # TODO(@themarwhal): Migrate to using full path for includes - GH8299
+    strip_include_prefix = "/orc8r/gateway/c/common/ebpf",
```

Comment:
we should be ok without this (we only need this for files not included into SessionD)

Reply:
Sessiond will use this lib in the next pr, so should I keep it?

Reply:
ooo then yeah keep it :)